codegenerator/cli/README.md

```diff
@@ -29,4 +29,6 @@ For support and updates, follow Envio on [Twitter](https://twitter.com/envio_ind
 
 Please create an [issue](https://github.com/enviodev/hyperindex/issues/new/choose) in our GitHub repository if you have specific suggestions or requirements. And a star is always appreciated 💫
 
+By submitting a Pull Request or making any contribution to this project, you automatically agree to our [Contributor License Agreement (CLA)](./licenses/CLA.md).
```

Comment:
This is the published package README. Should it also be added to the repository readme?

Reply:
Good question, sorry think this is actually not necessary, can probably leave this file as it was before

Reply:
Left it out